packages/storybook/src/ui/atoms/icons/icons.test.tsx

```diff
@@ -0,0 +1,487 @@
+    [<CommentIcon size="small" />],
+    [<CounterIcon size="small" status="secondary" value={1} />],
+    [<CrossIcon size="small" />],
+    [<EditIcon size="small" />],
+    [<EllipsisHorizontalIcon size="small" />],
+    [<EllipsisVerticalIcon size="small" />],
+    [<ExclamationIcon size="small" status="amber" />],
+    [<ExternalLinkIcon size="small" />],
+    [<FilterIcon size="small" />],
+    [<FolderPlusIcon size="small" />],
+    [<GridLayoutIcon size="small" />],
+    [<HamburgerIcon size="small" />],
+    [<InfoCircleIcon size="small" status="green" />],
+    [<LayerGroupIcon size="small" />],
+    [<ListLayoutIcon size="small" />],
+    [<PauseIcon size="small" />],
+    [<PlayIcon size="small" />],
+    [<PlusIcon size="small" />],
+    [<SpinnerIcon size="small" />],
+    [<StopIcon size="small" />],
+    [<TableIcon size="small" status="default" />],
+    [<ToggleOffIcon size="small" />],
+    [<ToggleOnIcon size="small" />],
+    [<TrashIcon size="small" />],
+    [<UserIcon size="small" />],
+    [<DotCircleIcon size="small" status="default" />],
+    [<SearchIcon size="small" />],
+  ])('Renders', (icon: JSX.Element) => {
+    render(icon)
+  })
+
+  it.each([
+    [<BullseyeIcon size="small" />, 'icon__bullseye'],
+    [<CalendarIcon size="small" />, 'icon__calendar'],
+    [<CheckIcon size="small" />, 'icon__check'],
+    [<ChevronCircleIcon size="small" direction="up" />, 'icon__chevron-circle'],
+    [<ChevronIcon size="small" direction="up" />, 'icon__chevron'],
+    [<CircleIcon size="small" status="default" />, 'icon__circle'],
+    [<ColumnIcon size="small" status="default" />, 'icon__column'],
+    [<CommentIcon size="small" />, 'icon__comment'],
+    [<CounterIcon size="small" status="secondary" value={1} />, 'icon__counter'],
+    [<CrossIcon size="small" />, 'icon__cross'],
+    [<EditIcon size="small" />, 'icon__edit'],
+    [<EllipsisHorizontalIcon size="small" />, 'icon__ellipsis__horizontal'],
+    [<EllipsisVerticalIcon size="small" />, 'icon__ellipsis__vertical'],
+    [<ExclamationIcon size="small" status="amber" />, 'icon__exclamation'],
+    [<ExternalLinkIcon size="small" />, 'icon__external-link'],
+    [<FilterIcon size="small" />, 'icon__filter'],
+    [<FolderPlusIcon size="small" />, 'icon__folder__plus'],
+    [<GridLayoutIcon size="small" />, 'icon__grid-layout'], // TODO: This is also inconsistent!
+    [<HamburgerIcon size="small" />, 'icon__hamburger'],
+    [<InfoCircleIcon size="small" status="green" />, 'icon__info-circle'],
+    [<LayerGroupIcon size="small" />, 'icon__layer__group'],
+    [<ListLayoutIcon size="small" />, 'icon__grid-layout'], // TODO: Surely this is wrong?
+    [<PauseIcon size="small" />, 'icon__pause'],
+    [<PlayIcon size="small" />, 'icon__play'],
+    [<PlusIcon size="small" />, 'icon__plus'],
+    [<SpinnerIcon size="small" />, 'icon__spinner'],
+    [<StopIcon size="small" />, 'icon__Stop'], // TODO: This is inconsistent
```

Comment:
@jsed500 @Moza805 Some of these names are very inconsistent - i.e. "icon__Stop". And one of them looks like a copypasta mistake (there are two "icon__grid-layout"). I know it's potentially dangerous to change them, but I'd quite like to (and fix any resulting errors in `ehr-client`). Thoughts?

Reply:
Yeah, it would be much nicer if they were standardized. The stop button has been brought in by me for the care plans work so it's fine if that breaks. grid-layout is only used for compact view toggle (I think).

Either way, our consuming react app doesn't use the class names anywhere yet so you should be fine to change these two without any issues